bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java

```diff
@@ -238,6 +274,24 @@ public void dispose() {
       super.dispose();
    }
 
+   private void enableControls() {
+
+      final boolean isAuthorized = StringUtils.hasContent(_labelAccessToken_Value.getText()) && StringUtils.hasContent(_labelRefreshToken_Value
+            .getText());
```

Comment:
Unlucky auto wrap, when this happens in my code, then I format it this way

![image](https://user-images.githubusercontent.com/1283445/137595425-3460ec96-f3c9-449a-8e35-430897fe3742.png)

Reply:
I need to remember to do it manually because Eclipse doesn't seem to be able to do it automatically.
Fixed

Reply:
I do too but I have found that sometimes using the "Clean up" formatting puts it back on 1 line